pkgs/games/redeclipse/default.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv, fetchFromGitHub, fetchurl, fetchpatch
+, curl, ed, pkgconfig, zlib, libX11
+, SDL2, SDL2_image, SDL2_mixer
+}:
+
+stdenv.mkDerivation rec {
+  pname = "redeclipse";
+  version = "1.6.0";
+
+  src = fetchurl {
+    url = "https://github.com/red-eclipse/base/releases/download/v1.6.0/redeclipse_1.6.0_nix.tar.bz2";
+    sha256 = "0j98zk7nivdsap4y50dlqnql17hdila1ikvps6vicwaqb3l4gaa8";
+  };
+
+  buildInputs = [
+    curl ed pkgconfig libX11 zlib
```

Comment:
Programs invoked during build like `pkgconfig` should go to `nativeBuildInputs`.

Reply:
Fixed